pkg/deploy/generator/scripts/gatewayVMSS.sh

```diff
@@ -67,6 +72,19 @@ include /etc/logrotate.d
     create 0600 root utmp
     rotate 1
 }
+
+# Maximum log directory size is 100G with this configuration
+# Setting limit to 100G to allow space for other logging services
+# copytruncate is a critical option used to prevent logs from being shipped twice
+${gateway_logdir} {
+    size 20G
+    rotate 5
+    create 0600 root root
+    copytruncate
+    nooldir
+    missingok
```

Comment:
What's the justification here? I think it may make sense to log errors in the case of missing log files.

Reply:
I believe logrotate will fail with an error, and not rotate the rest of the system logs. But I could be wrong it's been a while since I've had to troubleshoot logrotate. It can be finicky, and if it couldn't access the log for rotation I didn't want it to fail.  

> If the log file is missing, go on to the next one without issuing an error message.
The man page looks like it only issues an error and moves on, I'll remove that option.